docs/FAQ.md

```diff
@@ -3,49 +3,102 @@ title: eScriptorium Tutorial - FAQ
 
 ## Is eScriptorium accessible offline?
 
+If you use an instance of eScriptorium hosted on a web server, no, you cannot access eScriptorium offline.
+
+If you use a [local deployment](https://gitlab.com/scripta/escriptorium/-/wikis/docker-install) of eScriptorium (installed on your machine and hosted by a virtual server), yes, you can access eScriptorium offline.
+
 ## Can I synchronize a local eScriptorium application with a server?
 
+eScriptorium is a decentralized software, which means that there are as many databases as there are instances of eScriptorium. It is impossible to synchronise two instances, unless they share the same database, which is unlikely.
+
+You can, however, update the content of your projects and documents from one instance to another with the help of the [export](walkthrough_export.md) and [import](walkthrough_import.md) features. For this, you need to have an account on both the source and the target instances.
+
 ## Who can I contact to request a new feature or signal a bug?
 
+If you want to provide feedback on the application, you can contact the eScriptorium developper team with their [Gitter community](https://gitter.im/escripta/escriptorium). You can also open [tickets](https://gitlab.com/scripta/escriptorium/-/issues) in the Gitlab repository hosting the source code of the application. If you want to offer a bug fix or contribute to the project, you can check the ["Contributing" page](https://gitlab.com/scripta/escriptorium/-/wikis/contributing).
+
+!!! Note
+    If you use eScriptorium on a server, we recommand identifying and contacting the owners of the application/server before reporting a bug to the eScriptorium developpement team. They may have deactivated some already existing features or be aware of a bug caused by the infrastructure rather than the application itself.
+
 ## Where can I find more documentation in eScriptorium?
 
-A listing of known existing documentations on eScriptorium is available here: https://github.com/alix-tz/escriptorium-doc-finder
+A listing of known existing documentations on eScriptorium is available here: [escriptorium-doc-finder](https://github.com/alix-tz/escriptorium-doc-finder)
 
 It intends to cover documentation is all available languages. If you know of other ressources not listed there, you should consider adding it, even if you were not its creator.
 
 ## Is there a documentation in another language?
 
-A listing of known existing documentations on eScriptorium is available here: https://github.com/alix-tz/escriptorium-doc-finder
+A listing of known existing documentations on eScriptorium is available here: [escriptorium-doc-finder](https://github.com/alix-tz/escriptorium-doc-finder)
 
 It intends to cover documentation is all available languages. If you know of other ressources not listed there, you should consider adding it, even if you were not its creator.
 
 ## I made my own documentation for eScriptorium, where can I reference it?
```

Comment:
I think here, instead of the same response as above, you could say that the person can compare its documentation with the eScriptorium one and if something is missing in this tutorial compared to what they want, they can contribute (link to contribute again).